assets/svg_diagram_editor/circle.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
@rohitkatlaa looks like attribution is required for these images (see "Attribution is required." [here](https://www.flaticon.com/free-icon/rec_808569)).

Reply:
Based on [this](https://support.flaticon.com/hc/en-us/articles/207248209-How-I-must-insert-the-attribution-) page we should add attribution in the following manner:
> Insert the attribution on the page where the icon is shown. This can be placed next to the image or on the footer of the website.

Hm, I'm not sure if we can use them. @seanlip can you confirm?

Reply:
Given the context, I think that could be a bit messy. Can we create our own? Please consult with @mschanteltc.

Reply:
Hm, actually, can we use fontawesome or something? These seem like simple icons that should be available [here](https://fontawesome.com/icons). @rohitkatlaa, can you check?

Reply:
I have used font awesome icons, two polygon shapes are custom created using literallyCanvas.

Reply:
I have also added a pic of the updated UI in the description.